# [Mapping and quantifying mammalian transcriptomes by RNAseqr](https://www.nature.com/articles/nmeth.1226)

RNA-Seq is a technology that uses the deep sequencing technology to study transcriptome of an organism[1]. It can be used for differential gene and transcrip expression quantification, characterization of alternative splicing patterns, and gene fusion detection[2]. A general RNAseq workflow starts with RNA extraction, which is followed by ribosomal RNA depletion or mRNA enrichment, cDNA synthesis, preparation of a library and sequencing on a high-throuhput platform[3]. Then, reads are aligned on a reference genome or transcrips[4].


## Link Technology -> Application -> Statistics

RNAseq -> Differential Gene Expression -> quasi-likelihood F-tests

## Some RNAseq analysis workflow examples

[RNA-seq workflow: gene-level exploratory analysis and differential expression](https://bioconductor.org/packages/release/workflows/vignettes/rnaseqGene/inst/doc/rnaseqGene.html)

[RNAseq analysis in R prepared by Combine Australia](https://bioinformatics-core-shared-training.github.io/RNAseq-R/)

## References
1. [RNA-Seq: a revolutionary tool for transcriptomics](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2949280/pdf/nihms229948.pdf)
2. [RNA sequencing: advances, challenges and opportunities](https://www.nature.com/articles/nrg2934)
3. [RNA sequencing: the teenage years](https://www.nature.com/articles/s41576-019-0150-2)
4. [RNA-Seq: a revolutionary tool for transcriptomics](https://www.nature.com/articles/nrg2484)


## Members

| Name               | Github username |
|--------------------|-----------------|
| Bucher, Jonas      | @jonasbucherETH |
| Errera, Tiziana    | @tizerrer       |
| Karakulak, Tülay   | @KarakulakTulay |
| Müller, Rafael     | @molitoris      |
| Willimann, Melanie | @melimon8       |
| Zhiwei, Dou        |                 |
